.github/workflows/main.yml

```diff
@@ -84,6 +84,7 @@ jobs:
         env:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          GCP_APP_CREDENTIALS: ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS}}
```

Comment:
Needs to be mapped to GOOGLE_APPLICATION_CREDENTIALS

Reply:
Yup, the Google Python SDK will look for that environment variable and read it as a path to the JSON file or as a JSON string itself. 😉